cmd/dlt/dlt.go

```diff
@@ -0,0 +1,98 @@
+package dlt
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+)
+
+// InstallDLTSymlink creates a symlink named 'dlt' pointing to the real databricks binary.
+func InstallDLTSymlink() error {
+	path, err := exec.LookPath("databricks")
+	if err != nil {
+		return errors.New("databricks CLI not found in PATH")
+	}
+	realPath, err := filepath.EvalSymlinks(path)
+	if err != nil {
+		return fmt.Errorf("failed to resolve symlink: %w", err)
+	}
+
+	dir := filepath.Dir(path)
+
+	// Check write permission by trying to create and remove a temp file
+	testFile := filepath.Join(dir, ".permtest")
+	f, err := os.Create(testFile)
+	if err != nil {
+		return fmt.Errorf("no write permission in %s: %w", dir, err)
+	}
+	f.Close()
+	_ = os.Remove(testFile)
+
+	dltPath := filepath.Join(dir, "dlt")
+
+	// Check if 'dlt' already exists
+	if fi, err := os.Lstat(dltPath); err == nil {
+		if fi.Mode()&os.ModeSymlink != 0 {
+			target, err := os.Readlink(dltPath)
+			if err == nil && target == realPath {
+				return errors.New("dlt already installed")
```

Comment:
you can just return nil here instead of error and add a log message saying dlt is already installed